src/modules/admin/components/autoExit/AutoExitTable.tsx

```diff
@@ -0,0 +1,126 @@
+import { Box } from '@mui/material';
+import { useMatch, useNavigate } from 'react-router-dom';
+
+import AutoExitDialog from './AutoExitDialog';
+import NotCollectedText from '@/components/elements/NotCollectedText';
+import RouterLink from '@/components/elements/RouterLink';
+import { ColumnDef } from '@/components/elements/table/types';
+import DeleteMutationButton from '@/modules/dataFetching/components/DeleteMutationButton';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import HmisEnum from '@/modules/hmis/components/HmisEnum';
+import { AdminDashboardRoutes, Routes } from '@/routes/routes';
+import { HmisEnums } from '@/types/gqlEnums';
+import {
+  AutoExitConfigFieldsFragment,
+  DeleteAutoExitConfigDocument,
+  DeleteAutoExitConfigMutation,
+  DeleteAutoExitConfigMutationVariables,
+  GetAutoExitConfigsDocument,
+  GetAutoExitConfigsQuery,
+  GetAutoExitConfigsQueryVariables,
+} from '@/types/gqlTypes';
+import { evictAutoExitConfigs } from '@/utils/cacheUtil';
+import { generateSafePath } from '@/utils/pathEncoding';
+
+const columns: ColumnDef<AutoExitConfigFieldsFragment>[] = [
+  {
+    header: 'Length of Absence',
+    render: ({ lengthOfAbsenceDays }) => <>{lengthOfAbsenceDays} Days</>,
+  },
+  {
+    header: 'Project Type',
+    render: ({ projectType }) => (
+      <HmisEnum enumMap={HmisEnums.ProjectType} value={projectType} />
+    ),
+  },
+  {
+    header: 'Project',
+    render: ({ project }) =>
+      project ? (
+        <RouterLink
+          to={generateSafePath(Routes.PROJECT, { projectId: project.id })}
+        >
+          {project.projectName}
+        </RouterLink>
+      ) : (
+        <NotCollectedText variant='body2'>No Project</NotCollectedText>
```

Comment:
I think this wording is a bit more descriptive. The variant should be inherited so isn't needed explicitly
```suggestion
        <NotCollectedText>Any Project</NotCollectedText>
```